__tests__/app.test.js

```diff
@@ -161,6 +162,25 @@ describe('API', () => {
                     expect(article.created_at).toBe("2020-07-09T20:11:00.000Z");
                     expect(article.votes).toBe(100);
                     expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+                    expect(article.comment_count).toBe("11");
+                })
+        });
+        test('Status 200 - returns back existing article with index 2 and comment_count 0', () => {
+            return request(app)
+                .get("/api/articles/2")
+                .expect(200)
+                .then(({ body }) => {
+                    const article = body.article;
+
+                    expect(article.article_id).toBe(2);
```

Comment:
If you have already tested some of these properties previously I dont think you _**need**_ to test them here, but its totally fine :)